web/public/locales/en/common.json

```diff
@@ -508,32 +508,32 @@
-    "privacyDescription": "At _helpbuttons_, we respect your privacy and are committed to protecting your personal data. Our privacy policy outlines how we collect, store, use, and disclose your personal information. We comply with all relevant privacy laws and regulations, ensuring your data is used solely for the purpose of providing and improving our service. We are transparent about our data collection processes, and you have the right to access, update, or delete your personal data at any time. Remember, sharing personal information in _helpbuttons_ posts is at your discretion—be mindful and only share what you are comfortable with others knowing.\nPlease direct your questions to the following email: help@helpbuttons.org",
-    "securityDescription": "Your security is our top priority. At _helpbuttons_, we implement stringent security measures to protect the integrity of our software and the safety of our users. We continually update and review our systems to ensure they meet the highest standards of cybersecurity. However, we also believe that security is a shared responsibility. We encourage our users to practice safe online behaviors, like using strong passwords and protecting personal information.\nPlease direct your questions to the following email: help@helpbuttons.org",
-    "contactDescription": "Please direct your questions to the following email: help@helpbuttons.org",
-    "networksEthicsDescription": "",
+    "helpbuttonsHighlight": "Helpbuttons",
+    "helpbuttonsDescription": "is the open-source software that powers this app. A tool to support communities of any type and local groups in building their digital goods, coordinate them and be tecnologically autonomous. \n\n With Helpbuttons, you can create your own networks for sharing food, housing, transportation, or anything else you have in mind, and then federate with other networks to share users and resources. Helpbuttons specializes in connecting people who don't yet know each other, for various purposes, in the real world.",
+    "forDescription": "We aim to foster a cohesive society, promote cooperation, and optimize the design of collaborative tools in order to fully leverage the advantages of the internet. \n\nIn brief, we've observed the substantial amounts of money, time, and user participation required each time a new collaborative app is launched. This seems unnecessary. Most of these tools share a significant portion of their core functionalities. We believe these commonalities could be managed using the same open software, simply adapted for each unique purpose through various modules.",
+    "helpbuttonDescription": "A Helpbutton refers to a post. Every post published by a user in Helpbuttons is termed a helpbutton. Helpbuttons can represent offers, needs, exchanges, transport proposals, businesses, and so forth. They can also be renamed to suit the specific needs of your app.",
```

Comment:
are you sure about this? shouldn't it be _helpbutton_ ?

Reply:
yep